### PR TITLE
feat(Examples): Change Project.md feature for MAX32655, MAX32665, and MAX32690 

### PR DESCRIPTION
**Pull request by JC-282-AD**
### Description

1. BLE Examples: add feature to set advertising name through project.mk 
2. BLE Examples: add feature to configure security though project.mk 
3. BLE Examples: Allow user to select ME17 and ME14 chip version from project.mk and select PHY properly.

### CODE REVIEW

**Comment by EricB-ADI** [comment]
Can you add the advertising names update to otas and otac as well?

**Comment by JC-282-AD** [comment]
> Can you add the advertising names update to otas and otac as well?

Done! I also synced these updates to ME17. Since I don't have the board for ME17, I didn't test it, but I think it should be good. Just copy and paste for these same updates.

File: Examples/MAX32665/Bluetooth/BLE_datc/project.mk
[the remark this answers is not in the document]
Comment (JC-282-AD) [reply Examples/MAX32665/Bluetooth/BLE_datc/project.mk:1]: Done! My branch diverges somehow. I fix it and hopefully it is clean.

**Comment by JC-282-AD** [comment]
Merged

**Pull request closed without merging**